src/main/java/org/spongepowered/api/world/extent/BiomeArea.java

```diff
@@ -22,43 +22,50 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
  * THE SOFTWARE.
  */
-
 package org.spongepowered.api.world.extent;
 
-import com.flowpowered.math.vector.Vector3i;
+import com.flowpowered.math.vector.Vector2i;
+
 import org.spongepowered.api.world.biome.BiomeType;
 
 /**
- * A volume containing biomes.
+ * A two-dimensional buffer for {@link BiomeType} data. This buffer has no
+ * direct relation to the world and changes to it are not synchronized to the
+ * world.
  */
-public interface BiomeVolume {
+public interface BiomeArea {
 
     /**
      * Get an object representing the biome at the given position.
      *
-     * <p>While {@code position} is a 3-dimensional position, biomes in
-     * Minecraft are column-based (over the X and Z plane). Therefore, the biome
-     * for all the blocks in a certain column will all be the same and changing
-     * the biome of one block in a column will change the biome for the
-     * entire column.</p>
-     *
      * @param position The position
      * @return The biome
      */
-    BiomeType getBiome(Vector3i position);
+    BiomeType getBiome(Vector2i position);
```

Comment:
Because area is 2D and volume is 3D (and universe is arbitrary). Basically, don't add it unless it's requested and don't pretend to have 3D biomes unless you really have them. There's a good solution for adding 3D support over 2D without breaking compatibility, so I suggest not going any further than needed right now.